.github/workflows/image-push-master.yml

```diff
@@ -59,4 +59,24 @@ jobs:
           push: true
           tags: ghcr.io/${{ github.repository }}:latest-ocp
           file: Dockerfile.openshift
-          platforms: linux/arm64
```

Comment:
Here I realized in my last PR I assumed this build was arm, but it is for `latest-ocp`, so this also corrects that.